xbmc/network/upnp/UPnPServer.cpp

```diff
@@ -659,10 +660,10 @@ CUPnPServer::OnBrowseDirectChildren(PLT_ActionReference&          action,
                                     const PLT_HttpRequestContext& context)
 {
     CFileItemList items;
-    NPT_String parent_id = TranslateWMPObjectId(object_id, m_logger);
+    m_logger->info("Received Browse DirectChildren request for encoded object '{}' (plain value: '{}'), with sort criteria {}",
+                   object_id, DecodeObjectId(object_id).GetChars(), sort_criteria);
```

Comment:
You're decoding `object_id` twice. Please use a local variable for the decoded `object_id`.